processor/src/tasks/processGerbers/index.ts

```diff
@@ -69,31 +69,32 @@ export default async function processGerbers(
 
     await execEscaped(['mkdir', '-p', path.join(outputDir, 'images')])
 
-    const files = globule.find(path.join(inputDir, '**'), { dot: true })
-
-    const gerberTypes = findGerberFiles(files, path.join(inputDir, gerberDir))
-    let gerbers = Object.keys(gerberTypes)
-
-    let inputFiles
-    // XXX this is 5 due to whats-that-gerber matching non-gerber files and 5
-    // being a number that works for the projects currently on kitspace. it
-    // could cause problems with new projects and should be fixed in
-    // whats-that-gerber
-    // https://github.com/tracespace/tracespace/issues/357
-    if (gerbers.length < 5) {
-      if (plottedGerbers.gerbers.length === 0) {
+    let inputFiles: PlottedGerbers['inputFiles']
+    let gerbers: PlottedGerbers['gerbers']
+
+    if (plottedGerbers.gerbers.length === 0) {
```

Comment:
```suggestion
    if (kitsyaceYaml.gerbers != null || plottedGerbers.gerbers.length === 0) {
```

I think you are reversing this bug otherwise. It will use the plotted Gerbers even when it has a Gerber folder from kitspace.yaml.

Reply:
Do you know a repo I can use as a test case for this?

Reply:
I'm afraid not and I have also re-written all the tests for the s3 work already. Let's open an issue to add a test case for it.